chapter_15_advanced_forms.asciidoc

```diff
@@ -508,10 +317,48 @@ it as an application-layer constraint:
 $ pass:quotes[*python manage.py makemigrations*]
 Migrations for 'lists':
   src/lists/migrations/0005_alter_item_options_alter_item_unique_together.py
-    - Change Meta options on item
     - Alter unique_together for item (1 constraint(s))
 ----
 
+Now let's run the migration:
+
+[subs="specialcharacters,macros"]
+----
+$ pass:quotes[*python manage.py migrate*]
+----
+
+When you run the migration, you may encounter the following error:
+[subs="specialcharacters,macros"]
+----
+$ pass:quotes[*python manage.py migrate*]
+Operations to perform:
+  Apply all migrations: auth, contenttypes, lists, sessions
+Running migrations:
+  Applying lists.0005_alter_item_unique_together...Traceback (most recent call last):
+[...]
+sqlite3.IntegrityError: UNIQUE constraint failed: lists_item.list_id, lists_item.text
+
+[...]
+django.db.utils.IntegrityError: UNIQUE constraint failed: lists_item.list_id, lists_item.text
+----
+
+The problem is that we have at least one database record which used to be valid
+but after introducing our new constraint, the `unique_together`, it's no longer
+compatible.
+
+Just delete `src/db.sqlite3` and run the migration again. We can do this,
+because it's only a dev environment, and the data saved in the database isn't
+important.
```

Comment:
```suggestion
To fix this problem, we can just delete `src/db.sqlite3` and run the migration again.
We can do this because the database on our laptop is only used for dev, so the data in it is not important.
```